### PR TITLE
src/components/form: add component FormCreateInvoice for creating invoices

### PR DESCRIPTION
**Pull request by maceksimon**
Add component `FormCreateInvoice` for rendering a form for creating invoices as a company coordinator.
Form contains:
* summary of billing details
* link to edit billing details
* checkboxes for team members
* additional info fields
* option to enable donor entry fee

Add component `FormFieldCheckboxTeam` to render a single team with its members and implement checkbox logic.
Add composable `useFormatPrice` to render price labels based on given currency.
Rename variable `VatId` to more correct CompanyId (IČO) and use CompanyIdVat (DIČ).
Update types in `Organization.ts`
Add translations 
Add tests

Screenshot 2024-06-21 at 11 32 15

### CODE REVIEW

**Comment by tmszi** [comment]
Maybe better translation.

IČO - Business ID
DIČ - Tax ID

**Comment by tmszi** [comment]
form_create_invoice

**Review by tmszi — APPROVED** [review]

**Comment by maceksimon** [comment]
> Maybe better translation.
> 
> IČO - Business ID DIČ - Tax ID

Updated - used suggested names.

**Comment by maceksimon** [comment]
> form_create_invoice

Added translation string.

**Comment by tmszi** [comment]
Could you resolve conflicts, please?

File: src/components/form/FormCreateInvoice.vue [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+            <!-- Zip + city -->
+            <span v-if="organization.address?.zip">{{
+              organization.address.zip
+            }}</span
+            >&nbsp;<span v-if="organization.address?.city">{{
+              organization.address.city
+            }}</span>
+          </p>
+        </template>
+        <!-- Comapny ID -->
```
Comment (tmszi) [inline src/components/form/FormCreateInvoice.vue:113]: Fix comment please. 

`<!-- Business ID -->`

File: src/components/form/FormCreateInvoice.vue [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+        <p
+          class="q-mb-xs"
+          v-if="organization?.identificationNumber"
+          data-cy="form-create-invoice-organization-id"
+        >
+          {{ $t('form.labelBusinessId') }}:
+          {{ organization.identificationNumber }}
+        </p>
+        <!-- VAT ID -->
```
Comment (tmszi) [inline src/components/form/FormCreateInvoice.vue:122]: Fix comment please. 

`<!-- Tax ID -->`

File: src/components/form/FormCreateInvoice.vue (hunk 1 above)
Comment (maceksimon) [reply src/components/form/FormCreateInvoice.vue]: Updated.

File: src/components/form/FormCreateInvoice.vue (hunk 2 above)
Comment (maceksimon) [reply src/components/form/FormCreateInvoice.vue]: Updated.

**Pull request merged**